crates/controller/src/command.rs

```diff
@@ -162,6 +170,31 @@ fn left_click_handler(
         };
 
         let event = match pointer.entity().filter(|&e| playable.contains(e)) {
+            Some(entity) if is_double_click => {
+                let selected_entity_type = playable.get_component::<ObjectType>(entity).unwrap();
```

Comment:
There is already a very similar functionality in [`select_in_area()`](https://github.com/DigitalExtinction/Game/blob/main/crates/controller/src/areaselect.rs#L58). So I suggest to generalize that system so it can filter based on `Option<ObjectType>` (new part of `SelectInRectEvent`) and thus this system would only send `SelectInRectEvent` with a rectangle consisting of full screen.

Reply:
Thanks, I didn't see that function.